ethereum/contracts/Messages.sol

```diff
@@ -88,9 +89,10 @@ contract Messages is Getters {
             /// introduced accidentally later due to the nontrivial storage
             /// semantics of solidity.
             require(sig.guardianIndex < guardianCount, "guardian index out of bounds");
+            require(signatory != address(0));
```

Comment:
I agree that this check is sensible, however, it might be redundant given that the guardian set is stored using this [method](https://github.com/wormhole-foundation/wormhole/blob/6de97de242e57be4a0a8a0840f5a16bf4cd55bda/ethereum/contracts/Setters.sol#L17).